packages/union-component/src/components/DonationFundDropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { SelectHTMLAttributes, useEffect, useState } from 'react';
+import DonationDropdown from './DonationDropdown';
+import { getMembershipFunds, IFund } from './utils';
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+  setFundId: (fundId: string) => void;
+}
+
+const DonationFundDropdown: React.FC<Props> = (props) => {
+  const { setFundId } = props;
+  const [options, setOptions] = useState<IFund[]>([]);
+
+  useEffect(() => {
+    const getOptions = async () => {
+      const funds = await getMembershipFunds();
+
+      if (funds?.length) {
+        setOptions(funds);
+        setFundId(funds[0].id as string);
+      }
+    };
+
+    getOptions();
+  }, [setFundId]);
+
+  const defaultValue = options?.[0]?.id;
```

Comment:
Do you really need this line?
When you `setFundId(funds[0].id as string);` the value of the select doesn't change?